libs/elasticsearch/README.md

```diff
@@ -192,4 +192,58 @@ set_llm_cache(
 ```
 
 When overriding the mapping and the document building, 
-please only make additive modifications, keeping the base mapping intact.
\ No newline at end of file
+please only make additive modifications, keeping the base mapping intact.
+
+## Embeddings cache usage
+
+Store and temporary cache embeddings.
+
+Caching embeddings is obtained by using the [CacheBackedEmbeddings](https://python.langchain.com/docs/modules/data_connection/text_embedding/caching_embeddings),
+in a slightly different way than the official documentation.
```

Comment:
Can you add something on how (and maybe why) it is different?

Reply:
"In a different way" because the documentation always mentions only the from_bytes_store method, whereas we show that it can be instantiated directly as an argument of CacheBackedEmbeddings.